collector/src/main/java/io/prometheus/jmx/JmxScraper.java

```diff
@@ -3,15 +3,12 @@
 import java.io.IOException;
 
 import java.lang.management.ManagementFactory;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
No problem. My IDE has a tendency to collect together dependencies, but I'll uncollect them.